docs/developer-docs/latest/update-migration-guides/migration-guides/v4/plugin/enable-plugin.md

```diff
@@ -0,0 +1,48 @@
+---
+title: v4 plugin migration - Enable a plugin - Strapi Developer Docs
+description:
+canonicalUrl:
+prev: /developer-docs/latest/update-migration-guides/migration-guides/v4/plugin-migration.md
+---
+
+<!-- TODO: update SEO -->
+
+# v4 plugin migration: Enable a plugin
+
+!!!include(developer-docs/latest/update-migration-guides/migration-guides/v4/snippets/plugin-migration-intro.md)!!!
+
+<br/>
+
+::: callout
+A Strapi v3 plugin was enabled if it was manually installed or found in the `plugins` directory.
+
+In Strapi v4:
+
+- If a plugin is installed via the [automatic plugins discovery](/developer-docs/latest/plugins/plugins-intro.md#automatic-plugins-discovery) feature, it is automatically enabled.
+- While developing a local plugin, the plugin must explicitly be enabled in [the `./config/plugins.js` file](/developer-docs/latest/setup-deployment-guides/configurations/optional/plugins.md) of the Strapi application.
+:::
+
+Enabling a plugin in v4 should be done manually with the following procedure:
```

Comment:
a *local* plugin? Otherwise it doesn't have to be done manually because of the discovery feature

Reply:
Good catch! Thanks 🙏